automatic/cutepdf/update_helper.ps1

```diff
@@ -21,17 +21,63 @@ function GetETagIfChanged() {
   return $etag
 }
 
+function Compare-Hashes {
+param (
+	[string]$my_path = "$PSScriptRoot",
+	[string]$reg_chksum = '\bchecksum\b'
+)
+	$gud = $false
+	$current_checksum = (gi "$my_path\tools\chocolateyInstall.ps1" | sls $reg_chksum) -split "=|'" | Select -Last 1 -Skip 1
+	if ($current_checksum.Length -ne 128) { throw "Can't find current checksum" }
+	$remote_checksum  = Get-RemoteChecksum $url
+	if ($current_checksum -ne $remote_checksum) {
+		Write-Host 'Remote checksum is different then the current one, forcing update'
+		$gud = $true
+	}
+return $gud
+}
+
+function Get-PaddingNumber {
+	$tags= Invoke-WebRequest -Method Head -Uri $url -UseBasicParsing
+	$date = $tags | Foreach { $_.Headers["Date"] }
+	$lastChange = $tags | Foreach { $_.Headers["Last-Modified"] }
+	$ts = New-TimeSpan -Start $lastChange -End $date
+return $ts
+}
+
+function Get-FixedVersion {
```

Comment:
Please don't roll your own function fo creating fix versions, instead use the one in my helper [wormies-au-helpers](https://wormiecorp.github.io/Wormies-AU-Helpers/docs/functions/get-fixversion).
This repository already installs the package for this helper during the CI run.

Reply:
@AdmiringWorm 
I will look into adjusting this function maybe using your helper.

Reply:
@AdmiringWorm 
I had as you say rolled my own due to your "helper" requiring `AU_Force` to work correctly. I have since fixed this barrowing from @majkinetor hash check